scripts/update-contributors.pl

```diff
@@ -26,6 +27,16 @@
 my $SkippedEmptyLines = 0;
 my $SkippedBadLines = 0;
 
+# Brief display by default.
```

Comment:
Lack of `--help` is a separate bug that needs resolving if we care. Location determined to be "correct" at that time is likely to differ from both the current and suggested.

The options being added here are common / de-facto standard ones with well-known behaviour. Documenting that behaviour at all is just my preference rather than a true need.

Reply:
> Lack of `--help` is a separate bug that needs resolving if we care. Location determined to be "correct" at that time is likely to differ from both the current and suggested.

Why are you saying the above? I do not think the above assertions contradict or invalidate my change request in any way.
 
> The options being added here are common / de-facto standard ones with well-known behaviour. Documenting that behaviour at all is just my preference rather than a true need.

This change request is not about documentation. It is about moving a portion of the new code where it can be more easily found by script users.